adaptors/adaptor.go

```diff
@@ -1,23 +0,0 @@
-/*******************************************************************************
- * Copyright 2018 Okta, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- ******************************************************************************/
-
-package adaptors
-
-type Adaptor interface {
-	New() Adaptor
-	GetKey(jwkUri string)
-	Decode(jwt string, jwkUri string) (interface{}, error)
-}
```

Comment:
By moving this into the root `adaptor.go` file, what is the process if someone wants to create their own adaptor that uses something other than `lestrrat-go/jwx`

Reply:
I don't remember why I did this. Why would someone want to do that?

Reply:
The thought here is that if they have their own preferred library, or for some reason, requirements don't allow them to use `lestrrat-go/jwx`, this gives them the ability to use their own

Reply:
Yeah I honestly don't understand all of the requirements for this project, I just know how we are using it at my company, so I will defer to your judgment there.